test.rb

```diff
@@ -0,0 +1,21 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+l = lambda {|a| a }
+l.(x)
```

Comment:
__[rubocop]__
* Prefer the use of `lambda.call(...)` over `lambda.(...)`. [:ref](https://github.com/bbatsov/ruby-style-guide#proc-call)